src/main/java/org/broadinstitute/hellbender/tools/variantdb/CommonCode.java

```diff
@@ -1,29 +1,51 @@
 package org.broadinstitute.hellbender.tools.variantdb;
 
 import htsjdk.samtools.SAMSequenceDictionary;
+import htsjdk.variant.variantcontext.Allele;
+import htsjdk.variant.variantcontext.VariantContext;
 import htsjdk.variant.vcf.*;
+import org.apache.commons.lang.StringUtils;
+import org.broadinstitute.hellbender.tools.variantdb.arrays.RawArrayTsvCreator;
+import org.broadinstitute.hellbender.utils.genotyper.IndexedAlleleList;
 import org.broadinstitute.hellbender.utils.variant.GATKVCFConstants;
 import org.broadinstitute.hellbender.utils.variant.GATKVCFHeaderLines;
 
+import java.util.ArrayList;
 import java.util.HashSet;
+import java.util.List;
 import java.util.Set;
 
 //TODO rename this or get rid of it. a place holder for now
 public class CommonCode {
-    public static final String NORMX = "NORMX";
-    public static final String NORMY = "NORMY";
-    public static final String BAF = "BAF";
-    public static final String LRR = "LRR";
 
 
+    public static String getGTString(final VariantContext variant) {
```

Comment:
I'm so surprised this code doesn't exist anywhere?  the VCFWriter must be doing (largely) the same thing?

Reply:
yes, but it is very embedded in the actual writing of the output and in htsjdk which for some reason i think it's more difficult for us to modify to make it modular.